src/Integration/Binding/BindingController.cs

```diff
@@ -29,11 +30,11 @@ internal class BindingController : HostedCommandControllerBase, IBindingWorkflow
         private readonly IProjectSystemHelper projectSystemHelper;
 
         public BindingController(IHost host)
-            : this(host, null, null)
+            : this(host, null)
         {
         }
 
-        internal /*for testing purposes*/ BindingController(IHost host, IBindingWorkflowExecutor workflow, IProjectSystemHelper projectSystemHelper)
+        internal /*for testing purposes*/ BindingController(IHost host, IBindingWorkflowExecutor workflow)
```

Comment:
In cases when there's an instance specific interface injection (like in this case) I kept the internal constructor but in most of the other cases it was no longer needed.